build-tools/packages/build-cli/src/codeCoverageAnalysisTool/codeCoverageCli.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IADOCodeCoverageConstants } from "./ADO/index.js";
+import { postCodeCoverageSummary } from "./codeCoveragePr.js";
+
+/**
+ * Interface that reflects the type of object returned by the coverage summary method
+ */
+export interface CodeCoverageSummary {
+	/**
+	 * Message to be put in the comment
+	 */
+	commentMessage: string;
+
+	/**
+	 * Whether to fail the build or not
+	 */
+	failBuild: boolean;
```

Comment:
Will this actually fail the ADO PR build? Or be reported as a (new) failed check in Github? If the latter, then I'd rename for clarity.